This changes the default font from "arial" which we don't have in OpenIndiana to "LiberationSans-Regular".
--- gnuplot-6.0.5/term/gd.trm.orig
+++ gnuplot-6.0.5/term/gd.trm
@@ -1062,7 +1062,11 @@
 	if (external_default)
 		png_state.ttffont = gp_strdup(external_default);
 	else	/* Might as well try some plausible font; it's no worse than failing immediately */
+#ifndef sun
 		png_state.ttffont = gp_strdup("arial");
+#else /* sun */
+		png_state.ttffont = gp_strdup("LiberationSans-Regular");
+#endif /* sun */
 
 	free(png_state.default_ttffont);
 	png_state.default_ttffont = gp_strdup(png_state.ttffont);
